adding_a_checkbox.md

```diff
@@ -1,56 +1,93 @@
 # Adicionando uma CheckBox
```

Comment:
Can you add the number and emoji?
#19: 🔘 
Thk!

Reply:
Yes, I can.
I didn't add this before because of the other translations - they don't have any emojis.
But I'll add, no problem!

Reply:
Thanks!